zipline/data/loader.py

```diff
@@ -260,7 +269,7 @@ def ensure_treasury_data(symbol, first_date, last_date, now, environ=None):
     path.
     """
     loader_module, filename, source = INDEX_MAPPING.get(
-        symbol, INDEX_MAPPING['^GSPC'],
+        symbol, INDEX_MAPPING['SPY'],
```

Comment:
Not a blocker, but I wonder if we should move this default benchmark symbol out into a global somewhere.